griptape/drivers/image_generation/amazon_bedrock_image_generation_driver.py

```diff
@@ -1,16 +1,15 @@
 from __future__ import annotations
 
 import json
-from typing import Optional, TYPE_CHECKING, Any
-
+from typing import TYPE_CHECKING, Any
 from attr import define, field, Factory
-
 from griptape.artifacts import ImageArtifact
 from griptape.drivers import BaseMultiModelImageGenerationDriver
+
 from griptape.utils import import_optional_dependency
 
 if TYPE_CHECKING:
-    import boto3
+    import boto3  # type: ignore
```

Comment:
`import boto3` leaves us with this error:


> Stub file not found for "boto3"Pylance[reportMissingTypeStubs](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportMissingTypeStubs)

Very annoying, but it looks like it can be resolved by [boto3-stubs](https://pypi.org/project/boto3-stubs). Ignoring for now until we can play around with it.

Reply:
Looks like that package is just an install and we're good? Should we just knock that out now?

Reply:
Done